Quick links A/B test design history

rishilpatel97 opened the pull request.

adamsilver commented:
- Remove colons in headings
- Use gov style dates and times
- Avoid nesting bullet lists
- the screenshots are blurry
- the screenshots are not taken at 1200px wide

Bug:

![Screenshot 2022-08-31 at 15 09 04@2x](https://user-images.githubusercontent.com/37163/187699085-f924de1e-62fa-48b5-95b1-fc2cd8e47e23.png)

rishilpatel97 commented:
- removed colons in headings
- fixed bug
- used gov style dates
- keeping nested bullets
- replaced quick links screenshot with 1200px wide one
- cannot replace screenshot without quick links as this version does not exist any more
